Products/ZenUI3/browser/resources/js/zenoss/ExtOverrides.js

```diff
@@ -1098,4 +1098,50 @@ Ext.override(Ext.util.Sorter, {
         }
     });
 
+    Ext.data.Field.override({
+        constructor: function () {
```

Comment:
Override constructor of Ext.data.Field for Ext.data.field.Auto:
Write wraper fn for convert functions defined in model definitions. This will be done before data pass through Ext.data.reader.Reader.

- check if data in field is `object` and have `uid`'s if so add /cz/ prefix;
- if there custom `convert` function then call it;